domain/models/player.go

```diff
@@ -10,6 +10,6 @@ type Player struct {
     ID              int         `db:"id"`
     Name            string      `db:"name"`
     PlayingStageID  nulls.Int   `db:"playing_stage_id"`
-    CreateTime      time.Time   `db:"create_time"`
+    CreatedAt       time.Time   `db:"create_time"`
```

Comment:
> pop のAPIを使うと CreatedAt という名前のコラムに自動的に時間を入れてくれるが、そうでない場合は値を与えてないと空と見なし、検証段階で失敗するので、 DBの create_time コラムをうまく作動させるためにモデルで CreatedAt と結びつける必要がある。

こういうことなら，

```suggestion
    CreatedAt       time.Time   `db:"created_at"`
```

と変更して，`create_time` カラムの存在を消せば良いんじゃないかな？

Reply:
参考ドキュメント
- https://gobuffalo.io/en/docs/db/models/#define-a-simple-model

Reply:
`CreatedAt` という名前に意味はなく，`db:"..."` がきちんとリンクされている必要があります

Reply:
`pop` 側の実装で `CreatedAt` というフィールドをモデルの中にあるかを探して今の時間を入れている。逆に、`db:"..."` はこの `CreatedAt` のマッピング先で、名前はなんでも良いです。

https://github.com/gobuffalo/pop/blob/fcef263baf7d9bd39b186f21c90e1e9238e816cd/executors.go#L168

https://github.com/gobuffalo/pop/blob/fcef263baf7d9bd39b186f21c90e1e9238e816cd/model.go#L162

Reply:
> > pop のAPIを使うと CreatedAt という名前のコラムに自動的に時間を入れてくれるが、そうでない場合は値を与えてないと空と見なし、検証段階で失敗するので、 DBの create_time コラムをうまく作動させるためにモデルで CreatedAt と結びつける必要がある。
> 
> こういうことなら，
> 
> と変更して，`create_time` カラムの存在を消せば良いんじゃないかな？

もちろん、 スキーマーで `create_time` を `created_at` に変更しても良いが、マイグレーションする必要が出るし、うまく動作するから必要ないかと

Reply:
おお，誤読してたごめん．
つまり，モデル名構造体のメンバーで `CreatedAt` や `UpdatedAt` という名前で宣言すると，エンティティ作成時にいい感じに値を保持してくれる．`CreatedAt` などの後の `db:"..."` に db のカラムを指定すると，その値を入れてくれるってことで良いかな？？どうせ自動でやってくれるなら `UpdatedAt` も使いたいと思ってるから，その時に `create_time` のマイグレーションもすることにする！